src/main/java/io/th0rgal/oraxen/commands/PrintFontCommand.java

```diff
@@ -0,0 +1,111 @@
+
+                    } else printUnicode(commandSender, (String) args[0]);
+                }));
+    }
+
+    private void printGlyph(CommandSender sender, String glyphName) {
+        if (glyphName.equals("all")) {
+            int i = 0;
+            ComponentBuilder cb = new ComponentBuilder();
+            for (Glyph glyph : OraxenPlugin.get().getFontManager().getGlyphs()) {
+                cb.append(printClickableMsg("§r[" + ChatColor.GREEN + glyph.getName() + "§r] ", String.valueOf(glyph.getCharacter()), String.valueOf(glyph.getCharacter())));
+                if (i % 3 == 0) {
+                    sender.spigot().sendMessage(cb.create());
+                    cb = new ComponentBuilder();
+                }
+                i++;
+            }
+            if (cb.getParts().size() != 0)
+                sender.spigot().sendMessage(cb.create());
+            return;
+        }
+        Glyph g = OraxenPlugin.get().getFontManager().getGlyphs().stream().filter(glyph -> glyph.getName().equals(glyphName)).findAny().orElse(null);
+        sender.spigot().sendMessage(printClickableMsg(ChatColor.WHITE + g.getName(), String.valueOf(g.getCharacter()), "§r" + g.getCharacter()));
+    }
+
+    /**
+     * Parses code input to print a unicode list with decimal version of UTF-16
+     * the format is {hex unicode id}+{range to display} like this: "E000+10"
+     * @param sender command sender
+     * @param code unicode symbol with formatted range
+     */
+    private void printUnicode(CommandSender sender, String code) {
+        try {
+            char utf;
+            int range = 1;
+            if (code.matches("[A-Za-z0-9]{4}\\+[0-9]+")) {
+                String[] splitted = code.split("\\+");
+                utf = new String(Hex.decodeHex(splitted[0].toCharArray()), StandardCharsets.UTF_16BE).toCharArray()[0];
+                range = Integer.parseInt(splitted[1]);
+            } else {
+                Bukkit.getLogger().info(code);
+                utf = new String(Hex.decodeHex(code.toCharArray()), StandardCharsets.UTF_16BE).toCharArray()[0];
+            }
+            ComponentBuilder componentBuilder = new ComponentBuilder();
+            for (int i = 0; i < range; i++) {
+                componentBuilder.append(printClickableMsg(ChatColor.WHITE + "[" + ChatColor.AQUA + "U+" + Integer.toHexString(utf).toUpperCase() + "," + ((int) utf) + "(dec)" + ChatColor.WHITE + "] ",
+                        String.valueOf(utf),
+                        "§r" + utf));
+                if (i == 2) {
+                    sender.spigot().sendMessage(componentBuilder.create());
+                    componentBuilder = new ComponentBuilder();
+                }
+                utf++;
+            }
+            if (componentBuilder.getParts().size() != 0)
+                sender.spigot().sendMessage(componentBuilder.create());
+
+        } catch (DecoderException e) {
+            e.printStackTrace();
+        }
+    }
```

Comment:
not really sure what the purpose of this is? i assume this PR is meant to skip the point of adding the code into the  website or make it easier to get for people who dont set the code themselves. Personally think this can just be removed in favor of the above

Reply:
I mean it isnt really bad. You just grab the code you define (if you did so) and copy the unicode.
But I agree the process should be simpler. Just dont see any use for also sending the formatted unicode right here.

If you give players the glyphid, and a click will have them copy the unicode to clipboard is all

Reply:
do you mean the book feature? the problem is that the book is small and can only contain emojis and not guis because they are too large

Reply:
just mean remove the bottom msg there. Doesnt seem too useful imo when u can click just the glyphID to copy it directly and also hover to see it.

Think the codeportion above is for  that?

![image](https://user-images.githubusercontent.com/62521371/179015497-a5591877-7d27-4fbb-98b2-2545db0f4b7a.png)

Reply:
the last part "automates" the unicode website part in the creation of new glyphs

Reply:
Well if u insist it should remain im fine with it, just was confused on its purpose.
Since this would only really be useful if you didnt know your glyphcode as oraxen specified it for you.
but eh nitpick so